code-of-conduct.md

```diff
@@ -0,0 +1,147 @@
+## Quill Community Code of Conduct
+Quill exists to facilitate sharing content, by making it easy for content
+creators to publish and collaborate with others.
+
+Quill is a piece of technology, but more importantly, it is a community.
+
+We believe that our mission is best served in an environment that is friendly,
+safe, and accepting; free from intimidation or harassment.
+
+Towards this end, certain behaviors and practices will not be tolerated.
+
+### tl;dr
+
+- Be respectful.
+- We're here to help: abuse@quill.is
+- Abusive behavior is never tolerated.
+- Violations of this code may result in swift and permanent expulsion from the Quill community.
+- "Too long, didn't read" is not a valid excuse for not knowing what is in this document.
+
+### Table of Contents
+
+- [Scope](#scope)
+- [Friendly Harassment-Free Space](#friendly-harassment-free-space)
+- [Reporting Violations of this Code of Conduct](#reporting-violations-of-this-code-of-conduct)
+- [Copyright Violations](#copyright-violations)
+- [Consequences](#consequences)
+- [Addressing Grievances](#addressing-grievances)
+- [Contact Info](#contact-info)
+- [Changes](#changes)
+- [Credit and License](#credit-and-license)
+
+### Scope
+We expect all members of the Quill community to abide by this Code of Conduct at
+all times in all Quill community venues, online and in person, and in one-on-one
+communications pertaining to Quill affairs.
+
+This policy covers the usage of Quill public infrastructure, including the
+Quill.is HTTP gateways, as well as other Quill websites, Quill related events,
+and any other services offered by or on behalf of the Quill community. It also
+applies to behavior in the context of the Quill Open Source project communities,
+including but not limited to public GitHub repositories, Slack teams, IRC
+channels, social media, mailing lists, and public events.
+
+This Code of Conduct is in addition to, and does not in any way nullify or
+invalidate, any other terms or conditions related to use of Quill services.
+
+The definitions of various subjective terms such as "discriminatory", "hateful",
+or "confusing" will be decided at the sole discretion of the Quill abuse team.
```

Comment:
"Abuse team" seems to give the wrong impression. Maybe call this the "moderation" team instead?